### PR TITLE
Updates based on edge extensions tvOS support release

### PR DESCRIPTION
**Pull request by addb**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/documentation/identity-for-edge-network/release-notes.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Release Notes
 
+## February 28, 2023
+
+### iOS AEPEdgeIdentity 1.2.0
+
+* Added tvOS support.
+* Identity map no longer accepts identity item with empty string value for id
```
Comment (emdobrin) [inline src/pages/documentation/identity-for-edge-network/release-notes.md:8]: item -> items
also use period at the end of the sentence

File: src/pages/documentation/release-notes/index.md [hunk 2]
```diff
@@ -4,6 +4,23 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
...
+## February 28, 2023
+
+### iOS AEPEdge 1.6.0
+
+* Added tvOS support.
+
+### iOS AEPEdgeIdentity 1.2.0
+
+* Added tvOS support.
+* Identity map no longer accepts identity item with empty string value for id
```
Comment (emdobrin) [inline src/pages/documentation/release-notes/index.md:22]: same here

File: src/pages/documentation/identity-for-edge-network/release-notes.md (hunk 1 above)
Comment (cacheung) [reply src/pages/documentation/identity-for-edge-network/release-notes.md:8]: Also add a period at the end of the sentence.

File: src/pages/documentation/consent-for-edge-network/release-notes.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## March 1, 2023
+
+### iOS AEPEdgeConsent 1.1.0
```
[the remark this answers is not in the document]
Comment (addb) [reply src/pages/documentation/consent-for-edge-network/release-notes.md:5]: Yes for iOS we have used this format in the [past](https://github.com/AdobeDocs/aep-mobile-sdkdocs/blob/main/src/pages/documentation/release-notes/2022.md#ios-aepedgeconsent-101)

**Review by emdobrin — APPROVED** [review]

File: src/pages/documentation/identity-for-edge-network/release-notes.md [hunk 4]
```diff
@@ -1,5 +1,12 @@
 # Release Notes
 
+## February 28, 2023
+
+### iOS AEPEdgeIdentity 1.2.0
+
+* Added tvOS support.
+* Identity map no longer accepts identity items with an empty string value for id.
```
Comment (calebk1) [inline src/pages/documentation/identity-for-edge-network/release-notes.md:8]: "... for ID."

File: src/pages/documentation/release-notes/index.md [hunk 5]
```diff
@@ -4,6 +4,23 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
...
+## February 28, 2023
+
+### iOS AEPEdge 1.6.0
+
+* Added tvOS support.
+
+### iOS AEPEdgeIdentity 1.2.0
+
+* Added tvOS support.
+* Identity map no longer accepts identity items with an empty string value for id.
```
Comment (calebk1) [inline src/pages/documentation/release-notes/index.md:22]: "... for ID."

**Review by calebk1 — COMMENTED** [review]
Just a couple of comments! (I don't like using id when we can use ID.)

**Review by calebk1 — APPROVED** [review]
LGTM!

**Pull request merged**